docker/Dockerfile

```diff
@@ -33,14 +33,14 @@ RUN rustup self update \
   && rustup component add rustfmt
 
 RUN cargo fetch
-RUN mkdir -p /risingwave/bin/connector-node
 
 RUN cargo build -p risingwave_cmd_all --release --features "static-link static-log-level" && \
   mv /risingwave/target/release/risingwave /risingwave/bin/ && \
   cargo clean
 
-RUN cd risingwave-connector-node && mvn -B package -Dmaven.test.skip=true
-RUN tar -zxvf /risingwave/risingwave-connector-node/assembly/target/risingwave-connector-1.0.0.tar.gz -C /risingwave/bin/connector-node
+RUN cd /risingwave/java && mvn -B package -Dmaven.test.skip=true && \
+    mkdir -p /risingwave/bin/connector-node && \
```

Comment:
maybe should delete here.

Reply:
This is in the builder, so I think the tar file will not be included in the final image